docs/source/conf.py

```diff
@@ -22,23 +22,14 @@
 
 import sphinx_rtd_theme
 import git
-import yaml
 
 from sphinx_gallery.sorting import ExplicitOrder
 
 TMP_FOLDER = Path(__file__).parents[2] / "tmp"
 TMP_FOLDER.mkdir(exist_ok=True)
 
-# Generate a JSON compatibility table
-
-html_extra_path = []
-
-with open("additional/releases.yaml") as f:
-    compat_table = yaml.safe_load(f)
-    dest = Path(TMP_FOLDER, "releases.json")
-    with open(dest, "w") as f:
-        json.dump(compat_table, f)
-        html_extra_path.append(str(dest))
+with open("additional/releases.json") as f:
+    compat_table = json.load(f)
```

Comment:
In this setup we're still serving the release table from the substra doc rather than the repo directly, so I'd be in favor of keeping it in YAML simply because it's easier for humans to edit in my opinion. Let the machines deal with JSON ^^

Reply:
What do you mean bu serving from the repo directly? In every case, we'll need to have the file into the `build/html` directory. Plus, the content of the compatibility table is [needed](https://github.com/Substra/substra-documentation/blob/main/docs/source/conf.py#L47-L51) anyway.

Does not seem that big of a deal to maintain a `json` instead of a `yaml` even though I agree that `yaml` is more human-friendly.

Reply:
What I meant was instead of querying `docs.substra.org/en/stable/releases.json`, clients could instead query `github.com/Substra/substra-documentation/blob/main/docs/source/additional/releases.json`, freeing us from having to work it into the build systems. This IMO would be the biggest reason for switching it to JSON. But you provide a very good solution on this PR for putting it in the doc (which is more elegant, I feel) so let's forget about this.

I would vote for keeping it in YAML because we have to edit and review it by hand, but I don't feel strongly about this.

Reply:
Reverted.